app/pb/api.proto

```diff
@@ -219,6 +234,13 @@ message RepeatedStringValue {
   repeated string value = 2;
 }
 
+message RepeatedLanguageAbilityValue {
+  // A RepeatedLanguageAbilityValue is like a "protobuf wrapper" (like {Type}Value) but for repeated LanguageAbility
+  // allows us to determine if the user wanted to remote this list of LanguageAbility's
+  bool exists = 1;
+  repeated LanguageAbility value = 2;
```

Comment:
Very nitpick, but `repeated` fields should be plural, so `values`!

Reply:
I agree but keeping consistent w/ RepeatedStringValue, which also has `value` unfortunately

Reply:
OK